dao-smartcontract/test/Bounty.t.sol

```diff
@@ -218,6 +260,51 @@ contract BountyTest is Test {
         assertEq(token.balanceOf(user1), INITIAL_BALANCE + amount1 + amount2 + BOUNTY_AMOUNT);
     }
 
+    function test_claimableProposalAmount(uint256 bountyAmount, uint256 proposalAmount) public {
```

Comment:
I added some fuzz test here since bounty was set to zero. I can modify this if you want to keep unit tests in here

Reply:
Good work!